tests/lib/proc_utils.py

```diff
@@ -5,11 +5,29 @@
 """ Common utilities for working with processes.
 """
 
-import logging
 import subprocess
+import sys
+import logging
+
+
+def get_env_from(script):
```

Comment:
looks like it would be better to call the script before tool execution in scope of `subprocess` run

Reply:
Also it's unused now